pkg/scheduler/framework/session.go

```diff
@@ -254,8 +258,8 @@ func (ssn *Session) Pipeline(task *api.TaskInfo, hostname string) error {
 }
 
 //Allocate the task to the node in the session
-func (ssn *Session) Allocate(task *api.TaskInfo, hostname string) error {
-	if err := ssn.cache.AllocateVolumes(task, hostname); err != nil {
+func (ssn *Session) Allocate(task *api.TaskInfo, hostname string, podVolumes *volumescheduling.PodVolumes) error {
```

Comment:
why do you change this interface?

Reply:
> why do you change this interface?

k8s v1.19.6 has changed the interface `AssumePodVolume`, arguement `podVolumes` is added and stores pv info at one node

Reply:
we need a design doc for that; the API change may break backward compatibility.

Reply:
> we need a design doc for that; the API change may break backward compatibility.

It's invisible for users and has no influence in backward compatibility

Reply:
both Action & Plugins are plugable, we need to keep backward compatibility.